tools/ImportBuddy/source/ImportBuddy/TheDiscDb.MakeMkv/MakeMkvHelper.cs

```diff
@@ -51,19 +53,65 @@ public async Task WriteLogs(int driveIndex, string path, bool cleanLogs = true)
             }
         }
 
-        public async Task CleanLogs(string path, CancellationToken cancellationToken = default)
+        public async Task CleanLogs(int driveIndex, string path, CancellationToken cancellationToken = default)
         {
             var output = new List<string>();
             var lines = await this.fileSystem.File.ReadAllLines(path, cancellationToken);
             bool fileChanged = false;
-            foreach (var line in lines)
+            foreach (var originalLine in lines)
             {
-                if (string.IsNullOrWhiteSpace(line) || line.StartsWith("MSG") || line.StartsWith("DRV"))
+                // Remove empty lines from the output
+                if (originalLine == string.Empty)
                 {
                     fileChanged = true;
                     continue;
                 }
 
+                string? line = null;
+                if (originalLine.StartsWith("MSG"))
+                {
+                    var msgLine = MessageLogLine.Parse(originalLine);
+                    var (toRedact, replacement) = msgLine.Code switch
+                    {
+                        // The debug file path is usually in the user's home directory
+                        "1004" => (msgLine.Params[0], "file::///redacted/by/ImportBuddy"),
```

Comment:
Is having special strings we replace values with necessary? It seems like we could just remove the parts of the strings (replace with "") or just replace with some constant like "***" if we want to know where the redacted parts are later.

Reply:
I was writing a big comment explaining my reasoning, and ended up talking myself into agreeing that a simple constant like `***` works fine for redacting things.